package.json

```diff
@@ -60,7 +60,7 @@
       "gzip": "3.5 KB"
     },
     "dist/play-pen.js": {
-      "none": "37000 KB",
+      "none": "37200 KB",
```

Comment:
wow. I wonder what changed to cost so much. I don't think we had any big bumps in the other repos. I was expecting this to go down since we had so many deletions from @devvit/protos for post guidance and it's kind of a bummer.

Reply:
Not sure! There's also a discrepancy between the compressed sizes on github actions vs local build.  I had to bump the gzip'd size for play-pen.js although it passes locally.